exporter/convert.go

```diff
@@ -0,0 +1,52 @@
+package exporter
+
+import (
+	"strings"
+)
+
+type MetricValue struct {
+	Value  string
+	Labels map[string]string
+}
+
+func convertToMap(metrics []string) map[string]MetricValue {
+	matches := make(map[string]MetricValue, len(metrics))
+	/*
+		match metric format
+		slow_query_latency_us.p95.5=0
+		slow_query_latency_us{space=nba}.p95.5=0
+	*/
+	for _, metric := range metrics {
+		metric, label := splitMetric(metric)
+
+		s := strings.Split(metric, "=")
+		if len(s) != 2 {
+			continue
+		}
+		matches[s[0]] = MetricValue{Value: s[1], Labels: label}
+	}
+
+	return matches
+}
+
+// split slow_query_latency_us{space=nba}.p95.5=0 => slow_query_latency_us.p95.5=0, map[space:nba]
+func splitMetric(metric string) (string, map[string]string) {
+	start := strings.Index(metric, "{")
+	end := strings.LastIndex(metric, "}")
+
+	if start == -1 || end == -1 {
+		return metric, nil
+	}
+
+	label := make(map[string]string)
+	labelsStr := strings.Split(metric[start+1:end], ",")
```

Comment:
why the sep is ","

Reply:
becase the metric may like this `num_queries{space=nba,partition=1}`